src/NotifyQueueConsumer/Command/Model/SendToNotify.php

```diff
@@ -11,9 +11,9 @@ class SendToNotify
     protected string $filename;
     protected int $documentId;
     protected ?string $documentType;
-    protected ?string $recipientName;
-    protected ?string $recipientEmail;
-    protected ?string $sendBy;
+    protected string $recipientName;
+    protected string $recipientEmail;
+    protected string $sendBy;
```

Comment:
Ah I should have explained, we took a look at the `SendToNotifyHandler.php` and found that the functions that return those variables never return null so we refactored to reflect that.

Reply:
Might be worth revisiting this since [LPA also uses the queue consumer](https://github.com/ministryofjustice/opg-sirius/pull/5623/files) but doesn't set those fields.